src/Resources/config/services.xml

```diff
@@ -16,6 +16,9 @@
         <service id="webpack_encore.tag_renderer" class="Symfony\WebpackEncoreBundle\Asset\TagRenderer">
             <argument type="service" id="webpack_encore.entrypoint_lookup_collection" />
             <argument type="service" id="assets.packages" />
+            <argument type="collection">
+                <argument key="crossorigin" />
```

Comment:
this one should be removed. The default config should be an empty array (so `<argument type="collection" />`)

Reply:
Done.